yt_dlp/extractor/zdf.py

```diff
@@ -387,18 +389,19 @@ class ZDFChannelIE(ZDFBaseIE):
         'info_dict': {
             'id': 'planet-e',
             'title': 'planet e.',
+            'description': 'md5:87e3b9c66a63cf1407ee443d2c4eb88e',
         },
         'playlist_mincount': 50,
     }, {
         'url': 'https://www.zdf.de/gesellschaft/aktenzeichen-xy-ungeloest',
         'info_dict': {
             'id': 'aktenzeichen-xy-ungeloest',
-            'title': 'Aktenzeichen XY... ungelöst',
-            'entries': "lambda x: not any('xy580-fall1-kindermoerder-gesucht-100' in e['url'] for e in x)",
+            'title': 'Aktenzeichen XY... Ungelöst',
+            'description': 'md5:623ede5819c400c6d04943fa8100e6e7',
```

Comment:
Anything else I should verify in these tests? There is a whole bunch of per-entry data that might be worth testing, but I don't know how. The exact entries will certainly change frequently.

Reply:
if the entries change frequently then don't worry about it IMO